CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # Release Notes
 
+## Version 1.12.0 - 2020-07-09
+
+Features:
+
+* new addon : pod-security-policy [#8454](https://github.com/kubernetes/minikube/pull/8454)
+* new --extra-config option to config "scheduler" [#8147](https://github.com/kubernetes/minikube/pull/8147)
+
+ISO Changes:
+
+* Upgrade Docker, from 19.03.11 to 19.03.12 [#8643](https://github.com/kubernetes/minikube/pull/8643)
+* Upgrade crio to 1.18.2 [#8645](https://github.com/kubernetes/minikube/pull/8645)
+* Revert Buildroot version from 2020.02 to 2019.02 [#8649](https://github.com/kubernetes/minikube/pull/8649)
```

Comment:
these reverts don't need to be in the changelog

Reply:
sounds good, I think since we didn't publish the ISO no need to talk about the revert